caesars-palace/blocks/columns/columns.js

```diff
@@ -44,15 +45,4 @@ export default function decorate(block) {
       mqList.addEventListener('change', handleScreenChange);
     }
   }
-
-  const mediaQueryMatcher = window.matchMedia('(max-width: 768px)');
-  const handleScreenChange = (query) => {
-    if (query.matches) {
-      block.querySelectorAll('img').forEach((image) => {
-        image.closest('picture').replaceWith(createOptimizedPicture(image.src, image.alt, false, [{ width: '768' }]));
```

Comment:
Why did we need this here explictly again?

Reply:
The bug occurs only if using "replaceWith". I suspect it has to do something with the fact that replaeWith will do some cloning (hence we'll loose the original `picture` reference we got through the closure). I tried putting an id on the picture to grab it by id and move it, but a duplicate still managed to appear. So if we don't need all this, it's probably best to drop it.